src/logic/contracts/safeContractErrors.ts

```diff
@@ -36,6 +37,24 @@ export const getContractErrorMessage = async ({
     const contractOutput = abi.rawDecode(['string'], returnBuffer.slice(4))[0]
     return decodeMessage(contractOutput)
   } catch (e) {
-    return decodeMessage(e.message)
+    return null
```

Comment:
just to double-check if we're hiding the error on purpose

Reply:
If I understand correctly, this was causing the app to crash. It was being handled incorrectly and couldn't be decoded. @katspaugh, can you confirm?

Reply:
Yes, we weren't handling this in any special way because it's not critical and there's no good recovery strategy. Other than logging, which is a good move.